ckan/migration/versions/001_add_existing_tables.py

```diff
@@ -5,7 +5,7 @@
 
 
 def upgrade(migrate_engine):
-    meta = MetaData()
+    meta = MetaData(schema='public')
```

Comment:
please add a comment here, something like
`# state table conflicts with a table created by postgis docker image...`

Reply:
Good plan. Comment is now added.